teams.json

```diff
@@ -35,5 +35,6 @@
   "metamask/transaction-controller": "team-confirmations",
   "metamask/user-operation-controller": "team-confirmations",
   "metamask/multichain-transactions-controller": "team-sol,team-accounts",
-  "metamask/token-search-discovery-controller": "team-portfolio"
+  "metamask/token-search-discovery-controller": "team-portfolio",
+  "metamask/earn-controller": "team-stake"
```

Comment:
You might also want to update the `CODEOWNERS` file so that your team is free to review and merge PRs without the assistance of our team.

You can see how this was done for another new controller here: https://github.com/MetaMask/core/pull/5133/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7